packages/loader/container-loader/src/retriableDocumentStorageService.ts

```diff
@@ -42,16 +42,18 @@ export class RetriableDocumentStorageService implements IDocumentStorageService,
 		}
 		throw new Error("storage service not yet instantiated");
 	}
-	public get disposed() {
+	public get disposed(): boolean {
 		return this._disposed;
 	}
-	public dispose() {
+	public dispose(): void {
 		this._disposed = true;
 	}
 
 	public async getSnapshotTree(
 		version?: IVersion,
 		scenarioName?: string,
+		// API used below returns null
+		// eslint-disable-next-line @rushstack/no-new-null
```

Comment:
Nit, applicable in other places; I like the single line when possible, but don't feel strongly. Up to you.
```suggestion
		// eslint-disable-next-line @rushstack/no-new-null -- API used below returns null
```